core/scss/core/colors/index.scss

```diff
@@ -0,0 +1,43 @@
+///
```

Comment:
I put this file here because this is where it is found in the styleguide and where I would expect someone to go looking at first.